ystia/yorc
# Bugfix for GH-212 - SERVICE_IP_LOOKUP not working

# Pull Request description

## Description of the change
Call to function replaceServiceIPInDeploymentSpec was not saved in variable deploymentRepr so not given to k8s API

### What I did
I added an unmarshaling call into the create operation (maybe it could be done in a better way)

### How to verify it
Deploy a kubernetes component where 2 deployments are connected (for example grafana in yorc-a4c-plugin/tosca-sample) 

Check yaml of grafana deployment that env variable 'GRAPHITE_URL' is correctly set

### Description for the changelog

## Applicable Issues
